### ISSUE TITLE
Devtools output from browser -- Network profile

### ISSUE THREAD

**Issue by joe-farro** [xolvio/chimp#428]
#####  Expected behaviour
I am able to direct devtools output to a file(s). Of particular interest, to me, is the network profile tab. The ability to save a har would be nice.

#####  Actual behaviour
Not able to direct devtools output to a file.

##### Steps to reproduce the problem
Run chimp.

#####  Used version

* Chimp: 0.38
* Node.js: v4.4.6
* Operation system: Mac OSX 10.11.5

#####  Console / Log Output

n/a

------- 
*Join our Slack [xolv.io/community](http://xolv.io/community) #chimp channel, where you can find help and help others.*

**Comment by joe-farro** [comment]
Worth noting: https://bugs.chromium.org/p/chromedriver/issues/detail?id=1316

**Comment by tiffon** [comment]
This can be fixed with pull-request #429 by via the following `seleniumArgs` in a chimp config file:

```
{
    ...
    seleniumStandaloneOptions: {
        seleniumArgs: [
            '-Dwebdriver.chrome.logfile=/tmp/awesome.log',
            '-Dwebdriver.chrome.verboseLogging=True'
        ],
        ...
    }
}
```

Creates a log with great output; plenty of awesome info.
